cypress/integration/features/course/createCourse.feature

```diff
@@ -0,0 +1,13 @@
+Feature: Set of tests to create course
+
+    Background: User opens Schul-cloud homepage website
+        Given user arrives on the Schul-Cloud homepage
+
+    Scenario Outline: Adding a new course
+        Given I am logged in as a teacher
+        When '<userRole>' visit the rooms overview
+        Then '<userRole>' able to create a course in the old course creation page
+        Then '<userRole>' should be able to delete the course
```

Comment:
the feature we can simply write as follows IMO:

Feature: To add a new course by the teacher

    As a teacher I want to create a new course so that I can teach and provide online course related work to the students

Scenario: Adding a new course
        Given I am logged in as a teacher
        When I visit the rooms overview
        Then I am able to create a course in the old course creation page